broker-cli/commands/order/summary.js

```diff
@@ -15,8 +15,8 @@ function createUI (market, orders) {
   const unitWidth = Math.floor(windowWidth / 16)
 
   const orderTable = new Table({
-    head: ['Order ID', 'Side', 'Amount', 'Limit Price', 'Time', 'Status'],
-    colWidths: [5 * unitWidth, unitWidth, 3 * unitWidth, 3 * unitWidth, unitWidth, 2 * unitWidth],
+    head: ['Order ID', 'Status', 'Side', 'Amount', 'Limit Price', 'Time', 'Created At'],
+    colWidths: [2 * unitWidth, unitWidth, unitWidth, 3 * unitWidth, 3 * unitWidth, unitWidth, 3 * unitWidth],
```

Comment:
@treygriffith as an aside (the PR looks good), have we tried removing colWidths? I think that would force caporal to handle the column widths dynamically so we didn't need to calculate anything